antha/anthalib/mixer/mixer.go

```
@@ -197,6 +199,7 @@ func GenericMix(opt MixOptions) *wtype.LHInstruction {
 			}
 		}
 
+		solutions.UpdateComponentDetails(r.Results[0], opt.Components...) //nolint
```

Is there a *good* reason to ignore the error here?

---

they're warnings really, of the sort you mention above "No componentlist found for water, no concentration found for water so defaulting to 0.5 v/v water"